### PR TITLE
set DlgCoverArtFullSize window title only when window is shown

### PR DESCRIPTION
**Pull request by Be-ing**
This is an attempt to work around a GUI deadlock caused by an
old Xlib bug:
https://bugs.launchpad.net/mixxx/+bug/1789059
https://gitlab.freedesktop.org/xorg/lib/libx11/issues/25#note_50985

Unfortunately, I have not figured out how to reliably reproduce
the deadlock, so I am unsure if this actually works around the bug
in Xlib. Time will tell :/

### CODE REVIEW

**Comment by uklotzde** [comment]
Just one missing comment, otherwise LGTM.

File: src/library/dlgcoverartfullsize.cpp [hunk 1]
```diff
@@ -42,11 +42,11 @@ void DlgCoverArtFullSize::init(TrackPointer pTrack) {
     if (pTrack == nullptr) {
         return;
     }
-    slotLoadTrack(pTrack);
-
     show();
     raise();
     activateWindow();
+
+    slotLoadTrack(pTrack);
```
[the remark this answers is not in the document]
Comment (Be-ing) [reply src/library/dlgcoverartfullsize.cpp]: Good point. Fixed.

**Comment by Be-ing** [comment]
FWIW I don't notice any regressions with this. I looked to see if I could notice any flickering of the window title when the window is shown, but it seems to happen fast enough that I don't notice any flickering.

**Comment by uklotzde** [comment]
LGTM

**Pull request merged**